<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmInvestmentOperationsAdd</class>
 <widget class="QDialog" name="frmInvestmentOperationsAdd">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New investment operation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/document-edit.png</normaloff>:/xulpymoney/document-edit.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="wdgDatetime" name="wdgDT" native="true"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblType">
         <property name="text">
          <string>Operation type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbTiposOperaciones"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lblShares">
         <property name="text">
          <string>Number of shares</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQLineEdit" name="txtAcciones">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="wdgTwoCurrencyLineEdit" name="wdg2CCurrencyConversion" native="true"/>
     </item>
     <item>
      <widget class="wdgTwoCurrencyLineEdit" name="wdg2CPrice" native="true"/>
     </item>
     <item>
      <widget class="wdgTwoCurrencyLineEdit" name="wdg2CTaxes" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="wdgTwoCurrencyLineEdit" name="wdg2CComission" native="true"/>
       </item>
       <item>
        <widget class="QToolButton" name="cmdComissionCalculator">
         <property name="toolTip">
          <string>Comission calculator</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="wdgTwoCurrencyLineEdit" name="wdg2CGross" native="true"/>
     </item>
     <item>
      <widget class="wdgTwoCurrencyLineEdit" name="wdg2CNet" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="cmd">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../images/xulpymoney.qrc">
         <normaloff>:/xulpymoney/save.png</normaloff>:/xulpymoney/save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
  <customwidget>
   <class>wdgDatetime</class>
   <extends>QWidget</extends>
   <header location="global">wdgDatetime</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wdgTwoCurrencyLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">wdgTwoCurrencyLineEdit</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
